application/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ set(f3d_link_options_public "")
 
 # F3D_STRICT_BUILD
 list(APPEND f3d_compile_options_private ${f3d_strict_build_compile_options})
-if(F3D_STRICT_BUILD AND MSVC)
+if(MSVC)
```

Comment:
not specific to F3D_STRICT_BUILD anymore ? why

Reply:
Even if `F3D_STRICT_BUILD` is disabled, I don't see why we should not fix the warnings.  
`F3D_STRICT_BUILD` is just an option to allow developers to detect warnings easily.

Reply:
then this is not in the right location in the CMakeLists I think

Reply:
Let me revert this, it's unrelated to this PR anyway